contracts/extensions/token/erc20/ERC20TokenExtension.sol

```diff
@@ -156,6 +140,18 @@ contract ERC20Extension is AdapterGuard, IExtension, IERC20 {
         return bank.balanceOf(DaoHelper.TOTAL, tokenAddress);
     }
 
+    /**
+     * @dev Returns the amount of tokens assigned to all the members.
+     */
+    function totalAssignedTokens() public view returns (uint256) {
```

Comment:
There is a Slither error for this function. It would be nice to have a simple happy-path test for it.

Reply:
fixed